build.gradle

```diff
@@ -19,7 +19,6 @@ dependencies {
     implementation('org.jline:jline:3.25.0')
     implementation('org.json4s:json4s-native_2.13:3.6.12')
     implementation('org.ow2.asm:asm:9.6')
-    implementation('org.parboiled:parboiled_2.13:2.4.1')
```

Comment:
Can the scalactic dependency below also be removed?

Reply:
I don't know. I'm not sure what it does. A quick google tells me that it's related to ScalaTest and is some sort for collection of utility functions. It's not something to do with parsing though, so it should probably be removed in another PR. And someone other than me should make the call 😅

Reply:
Ok 👍